spec/models/user_spec.rb

```diff
@@ -0,0 +1,4 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+end
```

Comment:
There should never be a empty spec file, In the least test the creation of a user like this
```
  it 'should have valid Factory' do
    expect(FactoryBot.create(:user)).to be_valid
  end
```

Reply:
And we can also test for table columns in this spec